Python/cryptography/AES.py

```diff
@@ -0,0 +1,127 @@
+
+# Fortunately, we don’t need to implement AES from scratch
+# to do so We have to first install "pycryptodome" library
+# -- pip install pycryptodome
+
+from Crypto.Cipher import AES
+from Crypto.Hash import SHA256
+
+from Crypto import Random
+import base64
+
+# taking key from user and hashing it (can generate random key also )
+# hash key can be 16 bytes or 24 bytes or 32 bytes long
+key = input("Enter your secret key: ")
+# print(key)
+hash_key = SHA256.new(key.encode()).digest()
+print("Hashedn key for your secret key:")
+print(hash_key)
+len_hash_key = len(hash_key)
+# print(len_hash_key)
+
+# taking block size to be implemented in AES
+block_size = input("Enter the block-size in bytes: ")
+block_size = int(block_size)
+print(block_size)
+
+
+# Writing padding and unpadding functions
+# padding: as AES algorithm works in blocks of ciphers,our message gets divides into
+# blocks of ciphers which can be 128 or 192 or 256 bits long, we want our message in
+# multiples of this numbers. thats why we padd our msg.
+# unpadding can be used in time of decryption and padding in time of encryption
+# example: lets take string "I love cats" len-11 after padding "I love cats ----" len-16
+# padding character used here '-'
+
+
+def padding(msg, blocks_size):
+    padding_bytes_len = blocks_size - len(msg) % blocks_size
+# The chr() method returns a character from an integer (unicode)
+# we are generating padding char using padding_bytes_len
+# we can also use any padding char as explained in above example such as -,= etc
+    padded_char = chr(padding_bytes_len)
+    padding_string = padding_bytes_len*padded_char
+    padded_res = msg+padding_string
+    return padded_res
+
+
+def unpadding(msg):
+    last_char = msg[len(msg) - 1:]
+# The ord() function returns the number representing
+# the unicode code of a character.
+# as we added unicode code of character to make
+# it padded we are removing same.
+    unpadded_bytes_len = ord(last_char)
+    unpadded_res = msg[:-unpadded_bytes_len]
+    return unpadded_res
```

Comment:
Use proper Docstrings instead of putting comments for every line of code

Reply:
I have pushed requested changes. kindly check @HarshCasper